VAMobile/src/components/CommonErrorComponents/ErrorAlert.tsx

```diff
@@ -0,0 +1,50 @@
+import { ViewStyle } from 'react-native'
+import React, { FC } from 'react'
+
+import { AlertBox, Box, ButtonTypesConstants, VAButton, VAScrollView } from 'components'
+import { NAMESPACE } from 'constants/namespaces'
+import { useTheme } from 'utils/hooks'
+import { useTranslation } from 'react-i18next'
+
+export type ErrorAlertProps = {
+  /** Function called when the primary button is pressed */
+  onTryAgain: () => void
+  /** Title for the error AlertBox */
+  title?: string
+  /** Text to display in the error AlertBox  */
+  text: string
+}
+
+const ErrorAlert: FC<ErrorAlertProps> = ({ onTryAgain, title, text }) => {
```

Comment:
I feel like this might need to be better named along with the other error components at some point. I feel the name should be ErrorAlertRefresh or something like that to differentiate from the kinds of error alerts we have.

Reply:
All of our error alerts (except for `DowntimeError`) have an option for the refresh/tryAgain button, so adding `Refresh` to the end of the component name might not make it more distinctive. Maybe `ErrorAlertBox` would be more descriptive? I agree that our error components could use some tidying up. Might be worth creating a CU ticket for that